tests/cli/test_main.py

```diff
@@ -51,6 +51,12 @@ def test_param_override(self):
             assert cli.config.get('priority') == 'debug'
 
     def test_normal_run(self, capsys):
+        (flexmock(journal.Reader)
+            .should_receive('seek_tail')
+            .once())
+        (flexmock(journal.Reader)
+            .should_receive('get_previous')
+            .and_return({'__CURSOR': '0'}))
```

Comment:
Would be nicer not to have to put these in the tests, since we don't care about them behaviourally. See other notes for how to avoid it.

Reply:
Will do.